third_party/ti_simplelink_sdk/run_sdk_drivers_gen.py

```diff
@@ -0,0 +1,24 @@
+# Copyright 2020 Texas Instruments Incorporated
+
+"""A wrapper to run the SDK makefiles for Platform driver generation
+Args:
+1. [TI Simplelink SDK Root]
+2. [Matter repository Root]
+"""
+
+import os
+import subprocess
+import sys
+
+SDK_ROOT = sys.argv[1]
+CHIP_ENV_ROOT = sys.argv[2]
+
+if os.getenv('_PW_ACTUAL_ENVIRONMENT_ROOT'):
+    CHIP_ENV_ROOT = os.getenv('_PW_ACTUAL_ENVIRONMENT_ROOT')
+GCC_ARMCOMPILER_PATH = CHIP_ENV_ROOT + "/cipd/packages/arm/"
```

Comment:
can you do a check to see if this directory exist? An error of `.../packages/arm was not found` seems nicer than a make error.

Reply:
Thanks for the feedback, updated PR.